Common.mk

```diff
@@ -156,6 +156,7 @@ HELM_GIT_PATCH_TARGET?=$(HELM_SOURCE_REPOSITORY)/eks-anywhere-helm-patched
 ####################################################
 
 #################### BINARIES ######################
+GOLANG_VERSION?=""
```

Comment:
shouldnt need the quotes and you can remove them in the if below as well.

Reply:
Done.